style.css

```diff
@@ -0,0 +1,86 @@
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background: #ff1d58;
+ 
+}
+
+.container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+
+}
+
+.container a{
+    position: relative;
+    width: 160px;
+    height: 60px;
+    background: white;
+    display: inline-block;
+    margin: 20px;
+}
+
+.container a:before,
+.container a:after{
+    content: '';
+    position: absolute;
+    inset: 0;
+    background: red;
+    transform: 0.5s;  
+}
+
+.container a:nth-child(1):before,
+.container a:nth-child(1):after{
+background: linear-gradient(45deg, #00ccff, #ff1d58, #ff1d58, black);
+}
+
+.container a:nth-child(2):before,
+.container a:nth-child(2):after{
+background: linear-gradient(45deg, black, #ff1d58, #ff1d58,  #00ccff);
+}
+
+
+.container a:hover:before{
+    inset: -3px;
+}
+
+.container a:hover:after{
+    inset: -3px;
+    filter: blur(10px);
+}
+
+.container a span{
+    position: absolute;
+    top:0;
+    left: 0;
```

Comment:
grouping

Reply:
LGTM means?

Reply:
how do i group them?
i want it on top then at extreme left.

Reply:
Not that particular property, but in general I am saying. In whole stylesheet you have not done grouping.

Reply:
> LGTM means?

Looks good to me